### DISCUSSION TITLE
Upgrade Uavailable icon

### DISCUSSION THREAD

**Discussion by irazetti** [twilio-labs/paste#2887] [Request: Icon] [answered]
Hi!
I wanted to check if there is any recommendation for a scenario where we indicate that there is an "Upgrade available" in a badge.
See example:  
![upgrade](https://user-images.githubusercontent.com/104209586/207581023-8684eafd-19f2-4e8e-9d74-48dc6ca1085a.jpg)

**Comment by gloriliale** [comment]
Hi @irazetti! I notice that you're using the[ warning variant for Badge](https://paste.twilio.design/components/badge#warning)- is there a reason why the corresponding WarningIcon wouldn't work here?

**Comment by irazetti** [comment]
Hi!
Yeah, I was thinking that is not that "Dramatic" as a warning, because as long as long as the Flex UI version in the account supports the plugin version, it should be fine. So I think an icon that indicates that can fit better. 
Thanks!

**Comment by gloriliale** [comment, marked as answer]
Hi @irazetti, thanks for the context. It sounds like choosing a different variant for Badge might help with communicating your message: maybe the `new` variant with the NewIcon, or one of the decorative variants. I don't see a problem with using a decorative variant with the LoadingIcon necessarily, but I would question, since there isn't a clear choice for an icon, if one was needed here. 

Hope that guidance is helpful. We'll slow down our responses over the company break but will be available fully in the new year!
